spec/koans.js

```diff
@@ -86,40 +86,48 @@ describe('`string.includes()` finds string within another string. ', () => {
   describe('find a single character', function() {
     it('in a three char string', function() {
       /* const searchString = ???? */
-      //expect('xyz'.includes(searchString)).toBe(true);
+      const searchString =  "x"   //.includes("xyz");
+      expect('xyz'.includes(searchString)).toBe(true);
     });
     it('reports false if character was not found', function() {
       /* const expected = ????*/;
-      //expect('xyz'.includes('abc')).toBe(expected);
+      const expected = false
+      expect('xyz'.includes('abc')).toBe(expected);
     });
   });
 
   describe('find a string', function() {
     it('that matches exactly', function() {
+      const findSome = parameter => 'xyz'.includes();
       /* const findSome = .... => 'xyz'.includes();*/
-      //expect(findSome('xyz')).toBe(true);
+        // function findSome(par) {
+        // return "xyz".includes(par)
+        //   }
+      
+      const findSome = sentence.includes('z')
+      expect(findSome('xyz')).toBe(true);
     });
   });
```

Comment:
I am not sure what you are trying to do here, findSome is the name of a function, it is the same thing to write :+1: 
```js
const findSome = parameter => 'xyz'.includes(parameter);
```
and to write : 
```js
function findSome(par) {
   return "xyz".includes(par)
}
```
the point for this test was to understand how to use the includes method with one argument, but you know how to do that, you've done it before :)